### PR TITLE
Adding signups total

### PR DESCRIPTION
**Pull request by deadlybutter**
Still WIP - The caching had unintended consequences, need to make a separate index function that caches the response & leave the original untouched

This PR grabs the total signups from phoenix-ashes and adds it to the client store. Additionally it adds some caching on the sign ups index since it tends to take a few seconds to load. This PR also accounts for updating the total when a user signs up.

This pr _does not_ display anywhere, only visible in the dev console

screen shot 2017-04-11 at 11 30 54 am

(Note the number change below)

screen shot 2017-04-11 at 11 37 54 am

### CODE REVIEW

**Comment by deadlybutter** [comment]
Need to resolve conflicts but other than that this is ready for review!

**Comment by deadlybutter** [comment]
Also accidentally undid using the /total in the fender, ignore that for a sec, will be in next commit

**Comment by deadlybutter** [comment]
Ok, now this PR is ready 😅

File: app/Http/Controllers/SignupController.php [hunk 1]
```diff
@@ -31,6 +31,18 @@ public function index(Request $request)
...
 
     /**
+     * Get the total signups for this campaign.
+     *
+     * @param  int $campaignId
+     * @return \Illuminate\Http\Response
+     */
+    public function total($campaignId)
+    {
+        // Count = 1 is a hack to make the API actually return something. Don't ask.
```
Comment (DFurnes) [inline app/Http/Controllers/SignupController.php]: "Don't ask". The official tagline of Phoenix Ashes.

**Pull request merged**